include/drivers/lora.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2019 Manivannan Sadhasivam
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_LORA_H_
+#define ZEPHYR_INCLUDE_DRIVERS_LORA_H_
+
+/**
+ * @file
+ * @brief Public LoRa APIs
+ */
+
+#include <zephyr/types.h>
+#include <device.h>
+
+enum lora_signal_bandwidth {
+	BW_125_KHZ = 0,
+	BW_250_KHZ,
+	BW_500_KHZ,
+};
+
+enum lora_spreading_factor {
```

Comment:
why not using original's lora node API naming, so datarate: enum  lora_datarate ?
(Maybe that would be simpler for those who know/use lora node already)

Reply:
Hmm. I thought of following the exact convention used in hardware but you're right, it might be easier if we use 'datarate' for lora-node users.